apps/hash-graph/lib/authorization/tests/api.rs

```diff
@@ -68,10 +68,16 @@ impl TestApi {
 
                     let tuples: Vec<UntypedTuple> =
                         tuple_receiver.await.expect("failed to receive tuples");
-                    client
-                        .delete_relation(&tuples, [])
-                        .await
-                        .expect("failed to delete relations");
+
+                    if let Err(error) = client.delete_relations(&tuples, []).await {
+                        eprintln!(
```

Comment:
Shouldn't this be tracing::error instead?

Reply:
We don't have `tracing` set up in the tests and I don't want to mess around with that here.

Reply:
👍